configure.ac

```
@@ -7665,7 +7673,9 @@ PY_STDLIB_MOD([_curses_panel],
   [], [test "$have_panel" != "no"],
   [$PANEL_CFLAGS $CURSES_CFLAGS], [$PANEL_LIBS $CURSES_LIBS]
 )
-PY_STDLIB_MOD([_decimal], [], [], [$LIBMPDEC_CFLAGS], [$LIBMPDEC_LIBS])
+PY_STDLIB_MOD([_decimal],
+  [], [test "$have_mpdec" = "yes"],
```

@zware: this should make `configure` fail if mpdecimal is missing.

```suggestion
  [test "$have_mpdec" = "yes"], [],
```

---

This only marks the extension module as "disabled", so this won't work.